Header header
geometry_msgs/Pose[] pose
string[] uuid
float64[] roundness
float64[] bwratio
float64[] esterror
